modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -198,6 +198,64 @@ describe('browser.tinymce.core.EditorTest', () => {
     assert.equal(editor.getContent(), '<!-- x --><p>\u00a0</p>', 'setContent with comment bug #4409');
   });
 
+  it('TINY-7875: addContentFormatter', () => {
+    const editor = hook.editor();
+    let setCount = 0;
+    let getCount = 0;
+
+    const setFormatter = () => {
+      editor.setContent('set');
```

Comment:
It's a test function meant to put a distinctive value into the editor for further in the test, that's all.

And we probably don't want to block the further events if it's used like this, since it will be events with different formats, which may be handled differently by those who listen.

And no we can't. If we don't set the content we can't confirm it's been set as expected, and if we don't return a value we don't fit the format type.